### PR TITLE
Update ES config with heap size 1G, Update biothing studio build command

### PR DESCRIPTION
**Pull request by remoteeng00**
- ES service is randomly killed because out of memory issue.
- BIOTHINGS_REPOSITORY arg must be defined when build biothing-studio image.

### CODE REVIEW

File: ansible_playbook/configure.yml [hunk 1]
```diff
@@ -177,3 +177,9 @@
     - {"line": "", "re": "^cluster.initial_master_nodes.*$"}
     - {"line": "xpack.security.enabled: false", "re": "^xpack.security.enabled.*$"}
 
+- name: update /etc/elasticsearch/jvm.options again replacing min/max heap size
+  replace: dest=/etc/elasticsearch/jvm.options replace="{{ item.line }}" regexp="{{ item.re }}"
+  become: yes
+  with_items:
+    - {"line": "-Xms1g", "re": "^## -Xms4g$"}
+    - {"line": "-Xmx1g", "re": "^## -Xmx4g$"}
```
Comment (newgene) [inline ansible_playbook/configure.yml]: @remoteeng00 Is it OK to set it to a fixed 1g? Typically, we should set it to half of the available memory. Also it's slightly better to add a custom config file under `jvm.options.d` folder, instead of modifying `jvm.options` I think, base on the [documentation here](https://www.elastic.co/guide/en/elasticsearch/reference/current//advanced-configuration.html).

For reference, this is how we did in an ES cluster setup shell script. Of course, it's not for an ES within a docker container, so use it as you see fit.

```
# desired heap size (MB) to half of total memory
ES_HEAP_SIZE=$(grep MemTotal /proc/meminfo | awk '{printf "%d", $2/1024/2}')
# set heap size
cat > /etc/elasticsearch/jvm.options.d/heap.options << EOF
-Xms${ES_HEAP_SIZE}m
-Xmx${ES_HEAP_SIZE}m
EOF
```

Comment (remoteeng00) [reply ansible_playbook/configure.yml]: > it OK to set it to a fixed 1g?

I supposed the all-in-one docker container like Biothings Studio is for demo purpose, right?
And it will be used by a new guys (like me months before)
So I think it should run as persistent as possible, with the default heap size 4G, I got many failed when run Studio following document (on Mac 16G mem, 6GB mem for Docker)
So I think heap size 1g is a balancing number.

> Also it's slightly better to add a custom config file under jvm.options.d folder, instead of modifying jvm.options

basically, my idea is about minimize code change, so I choose this way, But I will update following your instruction.

Comment (newgene) [reply ansible_playbook/configure.yml]: Is default heap size 4G? I saw doc says it's half of available mem. Worth checking how much was set in docker by default (when you give 6G for Docker). It should be reported when ES was started.

If it's just for demo hub, yes, set to 1G is good enough, but for the generic Dockerfile, we cannot reply on a fix number like 1G. You can probably make it a customizable and optional parameter.

About the location of `heap.options`, yes, let's use the new way.

Comment (remoteeng00) [reply ansible_playbook/configure.yml]: Oops, my bad, it is not 4G. In my local, default value is 1966m. Anyway, I updated following your comments.
